discord/ext/commands/cog.py

```diff
@@ -202,6 +202,11 @@ def qualified_name(self):
         """:class:`str`: Returns the cog's specified name, not the class name."""
         return self.__cog_name__
 
+    @property
+    def command_attrs(self):
+        """:class:`dict`: Returns the commands attributes applied for the cog."""
+        return self.__cog_settings__
```

Comment:
```suggestion
        return types.MappingProxyType(self.__cog_settings__)
```

Needs the import of course.